clients/admin-ui/src/features/privacy-requests/constants.ts

```diff
@@ -10,7 +11,7 @@ export const SubjectRequestStatusMap = new Map<PrivacyRequestStatus, string>([
   [PrivacyRequestStatus.PENDING, "New"],
   [PrivacyRequestStatus.PAUSED, "Paused"],
   [PrivacyRequestStatus.IDENTITY_UNVERIFIED, "Unverified"],
-  [PrivacyRequestStatus.REQUIRES_INPUT, "Requires input"],
+  [PrivacyRequestStatus.REQUIRES_INPUT, "Requires Input"],
```

Comment:
we use sentence casing instead of title casing for pretty much everything. I think this should be reverted and above use "Awaiting email send"

Reply:
Cool I'll fix the other ones too

Reply:
fixed!